vagrant/vagrantXterm.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+xterm -e 'vagrant ssh' &
```

Comment:
Do a check:  If xterm exists, use it.  If Terminal.app exists, use it.  Otherwise, just open ssh in the current shell.